src/xenia/app/emulator_window.cc

```diff
@@ -174,10 +182,71 @@ bool EmulatorWindow::Initialize() {
   // FIXME: This code is really messy.
   auto main_menu = MenuItem::Create(MenuItem::Type::kNormal);
   auto file_menu = MenuItem::Create(MenuItem::Type::kPopup, L"&File");
+  // Recent Sub tap
+  auto recent_list = MenuItem::Create(MenuItem::Type::kPopup, L"&Open Recent");
+  //******Add Recent PATHS to menu***************************************
+  std::string LINES[10];
```

Comment:
Please follow naming conventions. This applies to all variables and members introduced by your changes. I won't make this comment for every instance.

Reply:
Variable LINES[10] should be lines[10]? Or recent_list to recent_menu?

Reply:
I read the naming conversion and made the changes. Could you check the code to let me know that there are no more instances of unconventional naming please?